packages/wonder-blocks-button/components/button.md

```diff
@@ -20,3 +20,127 @@ Disabled button example:
     disabled={true}
 >Label</Button>
 ```
+
+Button examples, `style` specified (Support width, position, margin, and flex styles):
+```js
+const {View} = require("wonder-blocks-core");
+<table>
+    <thead>
+        <tr>
+            <th style={{minWidth: '250px'}}>Styles</th>
+            <th style={{width: '100%'}}>Buttons</th>
+        </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td>width: '200px'</td>
+        <td>
+            <Button
+                onClick={(e) => console.log("Hello, world!")}
+                style={{width: '200px'}}
+            >Label</Button>
+        </td>
+    </tr>
+    <tr>
+        <td>width: '75%'</td>
+        <td>
+            <Button
+                onClick={(e) => console.log("Hello, world!")}
+                style={{width: '75%'}}
+            >Label</Button>
+        </td>
+    </tr>
+    <tr>
+        <td>display: 'block', margin: '0 auto'</td>
+        <td>
+            <Button
+                onClick={(e) => console.log("Hello, world!")}
+                style={{display: 'block', margin: '0 auto'}}
+            >Label</Button>
+        </td>
+    </tr>
+    <tr>
+        <td>flexGrow: 1</td>
+        <td>
+            <div style={{display: 'flex'}}>
+                <Button
+                    onClick={(e) => console.log("Hello, world!")}
+                    style={{flexGrow: 1}}
+                >Label</Button>
+            </div>
+        </td>
+    </tr>
+    <tr>
+        <td>flexShrink: 2, width: '300px'</td>
+        <td>
+            <div style={{display: 'flex'}}>
+                <Button
+                    onClick={(e) => console.log("Hello, world!")}
+                    style={{flexShrink: 2, width: '300px'}}
+                >Label</Button>
+                <div
+                    onClick={(e) => console.log("Hello, world!")}
+                    style={
+                        {
+                            width: '100%',
+                            background: '#d92916',
+                            textAlign: 'center',
+                            lineHeight: '40px',
+                            borderRadius: '4px',
+                            color: 'white',
+                        }
+                    }
+                >A wide div</div>
+            </div>
+        </td>
+    </tr>
+    <tr>
+        <td>alignSelf: 'flex-end'</td>
+        <td>
+            <div style={{display: 'flex'}}>
+                <Button
+                    onClick={(e) => console.log("Hello, world!")}
+                    style={{alignSelf: 'flex-end'}}
+                >Label</Button>
+                <div
+                    onClick={(e) => console.log("Hello, world!")}
+                    style={
+                        {
+                            background: '#d92916',
+                            textAlign: 'center',
+                            lineHeight: '100px',
+                            height: '100px',
+                            borderRadius: '4px',
```

Comment:
We shouldn't be advertising the ability to change the `height` or `borderRadius` as a feature since we'll be disallowing when we get around to statically type checking `style`.

Reply:
You can't change the height or border radius of a Button, but I wanted this element to show positioning.